pinguess/PinGuess.java

```diff
@@ -3,15 +3,51 @@
     //Variable Declaration
     
     //Method Declaration
+  
+      public String leftPad(Int number, Int targetLength) {
+        Int output = number + '';
+        while (output.length < targetLength) {
+            output = '0' + output;
+        }
+        return output;
+    }
+      
     //main is the function that runs first and starts the lifecycle
+
+      
     public static void main(String[] args) {
-        
-        String pinNumber = "5412";
-        
-        /*
-         Hint: Rewrite the leftPin Function from the js method into here
+       Boolean found = false;
+       Int count = 0;
+       String pinNumber = "5412";
+
+         while(!found && count < 10000){
+           if(check(count)){
+           found = true;
+            }
+           else count++
+        }
+         
          
-         */
+         If (found){
+         System.out.print("Your PIN is: " + pin);
+         }
     }
     
+    
+    public Boolean check(combo){
+      Int pinCode = Integer.parseInt(pinNumber);
```

Comment:
capitalization on int, as well as boolean (unless you mean to be using the Boolean class)